src/pages/registration/description_tab/DatePickerField.tsx

```diff
@@ -59,28 +59,23 @@ export const DatePickerField = () => {
       <DatePicker
         label={t('registration.description.date_published')}
         value={date}
-        PopperProps={{
-          'aria-label': t('registration.description.date_published'),
-        }}
         onChange={(newDate) => {
           updateDateValues(newDate, yearOnly);
           setDate(newDate);
         }}
-        inputFormat={yearOnly ? 'yyyy' : 'dd.MM.yyyy'}
+        format={yearOnly ? 'yyyy' : 'dd.MM.yyyy'}
         views={yearOnly ? ['year'] : ['year', 'month', 'day']}
         maxDate={new Date(new Date().getFullYear() + 5, 11, 31)}
-        mask={yearOnly ? '____' : '__.__.____'}
-        renderInput={(params) => (
-          <TextField
-            {...params}
-            data-testid={dataTestId.registrationWizard.description.datePublishedField}
-            variant="filled"
-            required
-            onBlur={() => !touchedYear && setFieldTouched(DescriptionFieldNames.PublicationYear)}
-            error={hasError}
-            helperText={hasError && errorYear}
-          />
-        )}
+        slotProps={{
+          textField: {
+            inputProps: { 'data-testid': dataTestId.registrationWizard.description.datePublishedField },
```

Comment:
Verifiser at ny plassering av data-testid ser greit ut for @Eiriknil før merge.

Vil helst ha de ett hakk lenger opp, men finner fader ikke ut hvordan man får den dit uten å legge inn ny parent selv, som jeg ikke har lyst til